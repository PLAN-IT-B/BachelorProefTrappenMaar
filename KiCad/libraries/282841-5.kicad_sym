(kicad_symbol_lib (version 20211014) (generator kicad_symbol_editor)
  (symbol "282841-5" (pin_names (offset 1.016)) (in_bom yes) (on_board yes)
    (property "Reference" "J" (id 0) (at -5.588 7.62 0)
      (effects (font (size 1.27 1.27)) (justify left bottom))
    )
    (property "Value" "282841-5" (id 1) (at -5.08 -10.16 0)
      (effects (font (size 1.27 1.27)) (justify left bottom))
    )
    (property "Footprint" "TE_282841-5" (id 2) (at 0 0 0)
      (effects (font (size 1.27 1.27)) (justify left bottom) hide)
    )
    (property "Datasheet" "" (id 3) (at 0 0 0)
      (effects (font (size 1.27 1.27)) (justify left bottom) hide)
    )
    (property "Comment" "282841-5" (id 4) (at 0 0 0)
      (effects (font (size 1.27 1.27)) (justify left bottom) hide)
    )
    (property "ki_locked" "" (id 5) (at 0 0 0)
      (effects (font (size 1.27 1.27)))
    )
    (symbol "282841-5_0_0"
      (rectangle (start -5.08 -7.62) (end 5.08 7.62)
        (stroke (width 0.254) (type default) (color 0 0 0 0))
        (fill (type background))
      )
      (pin passive line (at -10.16 5.08 0) (length 5.08)
        (name "1" (effects (font (size 1.016 1.016))))
        (number "1" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 2.54 0) (length 5.08)
        (name "2" (effects (font (size 1.016 1.016))))
        (number "2" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 0 0) (length 5.08)
        (name "3" (effects (font (size 1.016 1.016))))
        (number "3" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 -2.54 0) (length 5.08)
        (name "4" (effects (font (size 1.016 1.016))))
        (number "4" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 -5.08 0) (length 5.08)
        (name "5" (effects (font (size 1.016 1.016))))
        (number "5" (effects (font (size 1.016 1.016))))
      )
    )
  )
  (symbol "2_pin_connector" (in_bom yes) (on_board yes)
    (property "Reference" "U" (id 0) (at -5.08 5.08 0)
      (effects (font (size 1.27 1.27)))
    )
    (property "Value" "2_pin_connector" (id 1) (at 0 -6.35 0)
      (effects (font (size 1.27 1.27)))
    )
    (property "Footprint" "" (id 2) (at 0 6.35 0)
      (effects (font (size 1.27 1.27)) hide)
    )
    (property "Datasheet" "" (id 3) (at 0 6.35 0)
      (effects (font (size 1.27 1.27)) hide)
    )
    (symbol "2_pin_connector_0_0"
      (pin passive line (at -10.16 1.27 0) (length 5.08)
        (name "1" (effects (font (size 1.016 1.016))))
        (number "1" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 -1.27 0) (length 5.08)
        (name "2" (effects (font (size 1.016 1.016))))
        (number "2" (effects (font (size 1.016 1.016))))
      )
    )
    (symbol "2_pin_connector_0_1"
      (rectangle (start -5.08 3.81) (end 5.08 -3.81)
        (stroke (width 0.254) (type default) (color 0 0 0 0))
        (fill (type background))
      )
    )
  )
  (symbol "8_pin_connector" (in_bom yes) (on_board yes)
    (property "Reference" "U" (id 0) (at -5.08 12.7 0)
      (effects (font (size 1.27 1.27)))
    )
    (property "Value" "8_pin_connector" (id 1) (at 0 -13.97 0)
      (effects (font (size 1.27 1.27)))
    )
    (property "Footprint" "" (id 2) (at 0 -13.97 0)
      (effects (font (size 1.27 1.27)) hide)
    )
    (property "Datasheet" "" (id 3) (at 0 -13.97 0)
      (effects (font (size 1.27 1.27)) hide)
    )
    (symbol "8_pin_connector_0_0"
      (pin passive line (at -10.16 8.89 0) (length 5.08)
        (name "1" (effects (font (size 1.016 1.016))))
        (number "1" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 6.35 0) (length 5.08)
        (name "2" (effects (font (size 1.016 1.016))))
        (number "2" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 3.81 0) (length 5.08)
        (name "3" (effects (font (size 1.016 1.016))))
        (number "3" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 1.27 0) (length 5.08)
        (name "4" (effects (font (size 1.016 1.016))))
        (number "4" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 -1.27 0) (length 5.08)
        (name "5" (effects (font (size 1.016 1.016))))
        (number "5" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 -3.81 0) (length 5.08)
        (name "6" (effects (font (size 1.016 1.016))))
        (number "6" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 -6.35 0) (length 5.08)
        (name "7" (effects (font (size 1.016 1.016))))
        (number "7" (effects (font (size 1.016 1.016))))
      )
      (pin passive line (at -10.16 -8.89 0) (length 5.08)
        (name "8" (effects (font (size 1.016 1.016))))
        (number "8" (effects (font (size 1.016 1.016))))
      )
    )
    (symbol "8_pin_connector_0_1"
      (rectangle (start -5.08 11.43) (end 5.08 -11.43)
        (stroke (width 0) (type default) (color 0 0 0 0))
        (fill (type none))
      )
      (rectangle (start -5.08 11.43) (end 5.08 -11.43)
        (stroke (width 0) (type default) (color 0 0 0 0))
        (fill (type background))
      )
      (polyline
        (pts
          (xy -5.08 11.43)
          (xy -5.08 -11.43)
          (xy 5.08 -11.43)
          (xy 5.08 11.43)
          (xy -5.08 11.43)
        )
        (stroke (width 0.254) (type default) (color 0 0 0 0))
        (fill (type none))
      )
    )
  )
  (symbol "ITX0512SA" (in_bom yes) (on_board yes)
    (property "Reference" "PS1" (id 0) (at 0 -11.43 0)
      (effects (font (size 1.27 1.27)))
    )
    (property "Value" "ITX0512SA" (id 1) (at 0 -8.89 0)
      (effects (font (size 1.27 1.27)))
    )
    (property "Footprint" "libraries:Buck Converter" (id 2) (at -26.67 -6.35 0)
      (effects (font (size 1.27 1.27)) (justify left) hide)
    )
    (property "Datasheet" "" (id 3) (at 26.67 -7.62 0)
      (effects (font (size 1.27 1.27)) (justify left) hide)
    )
    (property "ki_keywords" "XP_POWER DC/DC isolated Converter module" (id 4) (at 0 0 0)
      (effects (font (size 1.27 1.27)) hide)
    )
    (property "ki_description" "XP Power 6W, 1000 VDC Isolated DC/DC Converter Module, Fully Regulated Single Output Voltage 12V, Â±500mA, 5V Input Voltage, SIP" (id 5) (at 0 0 0)
      (effects (font (size 1.27 1.27)) hide)
    )
    (property "ki_fp_filters" "*XP?POWER?ITXxxxxSA*" (id 6) (at 0 0 0)
      (effects (font (size 1.27 1.27)) hide)
    )
    (symbol "ITX0512SA_0_0"
      (pin power_in line (at -10.16 5.08 0) (length 2.54)
        (name "+Vin" (effects (font (size 1.27 1.27))))
        (number "1a" (effects (font (size 1.27 1.27))))
      )
      (pin power_in line (at -10.16 2.54 0) (length 2.54)
        (name "+Vin" (effects (font (size 1.27 1.27))))
        (number "1b" (effects (font (size 1.27 1.27))))
      )
      (pin power_in line (at -10.16 -2.54 0) (length 2.54)
        (name "-Vin" (effects (font (size 1.27 1.27))))
        (number "2a" (effects (font (size 1.27 1.27))))
      )
      (pin power_in line (at -10.16 -5.08 0) (length 2.54)
        (name "-Vin" (effects (font (size 1.27 1.27))))
        (number "2b" (effects (font (size 1.27 1.27))))
      )
      (pin power_out line (at 10.16 5.08 180) (length 2.54)
        (name "+Vout" (effects (font (size 1.27 1.27))))
        (number "3a" (effects (font (size 1.27 1.27))))
      )
      (pin power_out line (at 10.16 2.54 180) (length 2.54)
        (name "+Vout" (effects (font (size 1.27 1.27))))
        (number "3b" (effects (font (size 1.27 1.27))))
      )
      (pin power_out line (at 10.16 -2.54 180) (length 2.54)
        (name "-Vout" (effects (font (size 1.27 1.27))))
        (number "4a" (effects (font (size 1.27 1.27))))
      )
      (pin power_out line (at 10.16 -5.08 180) (length 2.54)
        (name "-Vout" (effects (font (size 1.27 1.27))))
        (number "4b" (effects (font (size 1.27 1.27))))
      )
    )
    (symbol "ITX0512SA_0_1"
      (rectangle (start -7.62 7.62) (end 7.62 -7.62)
        (stroke (width 0.254) (type default) (color 0 0 0 0))
        (fill (type background))
      )
      (polyline
        (pts
          (xy 0 -2.54)
          (xy 0 -3.81)
        )
        (stroke (width 0) (type default) (color 0 0 0 0))
        (fill (type none))
      )
      (polyline
        (pts
          (xy 0 0)
          (xy 0 -1.27)
        )
        (stroke (width 0) (type default) (color 0 0 0 0))
        (fill (type none))
      )
      (polyline
        (pts
          (xy 0 2.54)
          (xy 0 1.27)
        )
        (stroke (width 0) (type default) (color 0 0 0 0))
        (fill (type none))
      )
      (polyline
        (pts
          (xy 0 5.08)
          (xy 0 3.81)
        )
        (stroke (width 0) (type default) (color 0 0 0 0))
        (fill (type none))
      )
    )
  )
)
