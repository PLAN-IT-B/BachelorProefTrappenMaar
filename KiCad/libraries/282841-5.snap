{"u_token":"ogFD0bzdqrCJGVwJpo0Vt6QCo8V8dNhF","part_number":"282841-5","manufacturer":"TE Connectivity","mode":"place-part","plugin":".snap","source":"https://www.snapeda.com"}